### PR TITLE
PlayFair Cipher in Java

### PR DESCRIPTION
**Pull request by parjanyaacoder**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented Playfair Cipher Algorithm in Java

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x} I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Solves #519

### CODE REVIEW

**Comment by parjanyaacoder** [comment]
@SKAUL05  Please review.

File: Java/cryptography/playfairCipher.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import java.util.*;
```
Comment (atarax665) [inline Java/cryptography/playfairCipher.java:1]: Add description of Playfair Cipher in commented form.

File: Java/cryptography/playfairCipher.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+class Main
+{  
+   static int tellRow(char a[][] , char ch)
```
Comment (atarax665) [inline Java/cryptography/playfairCipher.java:4]: Use proper comments to explain what each function does. And use any [Java beautifier](https://codebeautify.org/javaviewer) to format your code.

**Review by atarax665 — DISMISSED** [review]

**Comment by atarax665** [comment]
Closing the PR as there's no response from contributor's end.

**Pull request closed without merging**